azure/templates/app_alerts.json

```diff
@@ -110,8 +110,8 @@
         "enabled": "[parameters('enableAlerts')]",
         "description": "[concat('Alert when average CPU utilization for ', variables('appServicePlanName'), ' is greater than 80%.')]",
         "severity": 1,
-        "evaluationFrequency": "PT1M",
-        "windowSize": "PT5M",
+        "evaluationFrequency": "PT5M",
```

Comment:
It would be nice to have this configurable per environment. We probably care less about issues like this in dev than we do in prod

Reply:
I think for these type of alerts we probably want to know just as much on development as we do on production - something pegging the CPU at 100% or throwing a bunch of HTTP 500 errors is just as crucial.